noble/assetlist.json

```diff
@@ -0,0 +1,66 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "noble",
+  "assets": [
+    {
+      "description": "The permissioned staking asset for Noble Chain",
+      "denom_units": [
+        {
+          "denom": "ustake",
+          "exponent": 0,
+          "aliases": [
+            "microstake"
+          ]
+        },
+        {
+          "denom": "stake",
+          "exponent": 6
+        }
+      ],
+      "base": "ustake",
+      "name": "Stake",
+      "display": "stake",
+      "symbol": "STAKE"
+    },
+    {
+      "description": "Frienzies are an IBC token redeemable exclusively for a physical asset issued by the Noble entity.",
+      "denom_units": [
+        {
+          "denom": "ufrienzies",
+          "exponent": 0,
+          "aliases": [
+            "microfrienzies"
+          ]
+        },
+        {
+          "denom": "frienzies",
+          "exponent": 6
+        }
+      ],
+      "base": "ufrienzies",
+      "display": "Frienzies",
+      "name": "frienzies",
+      "symbol": "FRNZ"
+    },
+    {
+      "description": "USD Coin",
+      "denom_units": [
+        {
+          "denom": "uusdc",
+          "exponent": 0,
+          "aliases": [
+            "microusdc"
+          ]
+        },
+        {
+          "denom": "usdc",
+          "exponent": 6
+        }
+      ],
+      "base": "uusdc",
+      "display": "usdc",
+      "name": "USD Coin",
+      "symbol": "USD"
```

Comment:
Oh, also, I believe the symbol should be 'USDC'

Reply:
Good catch 🙏